src/Lecture2.hs

```diff
@@ -191,7 +215,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] xs = xs
+merge xs [] = xs
+merge (x:xs) (y:ys)
+  | x > y = y : merge (x:xs) ys
```

Comment:
You can implement this function in a slightly more efficient way by having a separate case when both elements in the heads are equal. With this approach, you'll be able to shorten lists faster 🐴